dss-api.yml

```diff
@@ -702,6 +702,118 @@ paths:
                     enum: [unhandled_exception, illegal_arguments, bundle_already_exists]
                 required:
                   - code
+  /bundles/{uuid}/checkout:
+    post:
+      summary: Check out a bundle to HCA-owned cloud storage
+      description: |
+        User makes a request for a bundle version to be extracted. Storage system makes an attempt to validate
+        that the bundle UUID with an optionally provided version exists in a given replica.
+        If bundle failed to be located, the error is immediately raised to the user.
```

Comment:
"If the storage service fails to locate the bundle..."

Reply:
fixed